galleries/tutorials/pyplot.py

```diff
@@ -26,7 +26,7 @@
 # In :mod:`matplotlib.pyplot` various states are preserved
 # across function calls, so that it keeps track of things like
 # the current figure and plotting area, and the plotting
-# functions are directed to the current axes (please note that "axes" here
+# functions are directed to the current Axes (please note that "axes" here
 # and in most places in the documentation refers to the *axes*
 # :ref:`part of a figure <figure_parts>`
 # and not the strict mathematical term for more than one axis).
```

Comment:
This paragraph is slightly meta here, and seems like we may want to stay capalized for  more of this sentence, especially the quoted instance.

Reply:
I've reworded the paragraph.